source/common/config/filesystem_subscription_impl.cc

```diff
@@ -55,6 +55,8 @@ void FilesystemSubscriptionImpl::refresh() {
     stats_.update_time_.set(DateUtil::nowToMilliseconds(api_.timeSource()));
     stats_.version_.set(HashUtil::xxHash64(message.version_info()));
     stats_.version_text_.set(message.version_info());
+    if (message.has_control_plane())
+        stats_.control_plane_.set(message.control_plane().identifier());
```

Comment:
I assumed that by `DiscoveryResponse.control_plane`, we mean the identifier of the control plane.